website/docs/d/cloud_service_epg.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+subcategory: - "Cloud"
+layout: "aci"
+page_title: "ACI: aci_cloud_service_epg"
+sidebar_current: "docs-aci-data-source-cloud_service_epg"
+description: |-
+  Data source for ACI Cloud Service EPG
+---
+
+# aci_cloud_service_epg #
+
+Data source for ACI Cloud Service EPG
+Note: This resource is supported in Cloud APIC only.
+
+## API Information ##
+
+* `Class` - cloudSvcEPg
+* `Distinguished Name` - uni/tn-{tenant_name}/cloudapp-{application_name}/cloudsvcepg-{name}
+
+## GUI Information ##
+
+* `Location` - Application Management -> EPGs
+
+
+## Example Usage ##
+
+```hcl
+data "aci_cloud_service_epg" "example" {
+  cloud_applicationcontainer_dn  = aci_cloud_applicationcontainer.example.id
+  name                           = "example"
+}
+```
+
+## Argument Reference ##
+
+* `cloud_applicationcontainer_dn` - (Required) Distinguished name of the parent Cloud Application Container object.
+* `name` - (Required) Name of the Cloud Service EPG object.
+
+## Attribute Reference ##
+* `id` - (Read-Only) Attribute id set to the Dn of the Cloud Service EPG.
+* `annotation` - (Read-Only) Annotation of the Cloud Service EPG object.
+* `name_alias` - (Read-Only) Name Alias of the Cloud Service EPG object.
+* `access_type` - (Read-Only) This refers to the type of connectivity to the service. It could be a public or private connectivity. Allowed values are "Private", "Public", "PublicAndPrivate", "Unknown", and default value is "Public". Type: String.
```

Comment:
read only does not require allowed or default values

comment applies to all read-only attributes

```suggestion
* `access_type` - (Read-Only) This refers to the type of connectivity to the service. It could be a public or private connectivity. Type: String.
```